docs/source/run/install.rst

```diff
@@ -0,0 +1,139 @@
+recommended way to install Sopel from pypi.org is to perform the following
+command::
+
+    python3 -m pip install sopel
+
+This will ensure that Sopel is installed for the Python interpreter you want to
+use. You do not need to have root access to install Sopel. To check that Sopel
+is indeed installed, you can run the following line::
+
+    python3 -c "import sopel; print(sopel.__version__)"
+
+This will show the installed version of Sopel for this interpreter.
+
+When a new version of Sopel is released, you can update your install by reusing
+the same command with the ``-U`` option (short for ``--upgrade``)::
+
+    python3 -m pip install -U sopel
+
+You must restart Sopel for the update to take effect.
+
+.. __: https://pypi.org/project/sopel/
+
+Installing from Source
+======================
+
+For development purposes or to test a future release, you may want to install
+Sopel directly from its `source repository`__ on GitHub. To install from
+source, first select the version you want to install, and refer to the
+``README.rst`` file for instructions.
+
+We strongly recommend to use a virtualenv or other isolation mechanism so as to
+prevent any conflicts with your system's Python interpreter and libraries.
+
+.. warning::
+
+    Although possible, source installations are not supported. If you install
+    from sources, you may encounter unexpected bugs and unstable behaviors, and
+    we fully expect you to either find a solution by yourself, revert to a
+    previous version, or wait for a fix without an ETA.
+
+    However, we do appreciate :ref:`bug reports<faq-bug-report>` (with logs and
+    configuration details) and feedback. In that case,
+    :ref:`reaching out to us on IRC<faq-contact-us>` is the best approach.
+
+.. __: https://github.com/sopel-irc/sopel
+
+
+First run
+=========
+
+Once Sopel is installed, you should have access to a ``sopel`` command, as well
+as :doc:`other commands <cli>`.
+
+.. note::
+
+    By default, ``pip`` will install the command in ``~/.local/bin/``, which
+    might not be on your ``PATH``. You will need to add this folder to your
+    ``PATH`` for your shell to see the command.
+
+    For example, you could add this to your ``~/.profile`` file (works on
+    Ubuntu 22.04)::
+
+        # set PATH so it includes user's private bin if it exists
+        if [ -d "$HOME/.local/bin" ] ; then
+            PATH="$HOME/.local/bin:$PATH"
+        fi
+
+    Refer to your operating system's documentation for more information on how
+    to configure your ``PATH``.
+
+Initial Configuration
+---------------------
+
+To run the bot, you need a :doc:`configuration file <configuration>`, which you
+can create with the following command::
+
+    sopel configure
+
+This will run a wizard that helps you create a configuration file. It will
+ask you questions to fill in the details. When you see something in ``[square
+brackets]``, that's the default setting, and you can just hit Enter to keep
```

Comment:
```suggestion
brackets]``, that's the default setting, and you can just hit :kbd:`Enter` to keep
```

Only because I loved so much what `:kbd:` did elsewhere… And it looks good here too:

<img width="150" src="https://github.com/sopel-irc/sopel/assets/164140/a49e078a-b081-4473-b0f4-ea3b0d2cd7ca">

This is also very nitpicky.

Reply:
> This is also very nitpicky.

But so much better!